src/System.Private.ServiceModel/tests/Stress/SharedPoolsOfWCFObjects/TestHelpers.cs

```diff
@@ -124,38 +205,37 @@ public CommunicationObjectEventVerifier(ICommunicationObject channel)
             }
         }
 
-        private static long s_closeCalls = 0;
         public static void CloseChannel<C>(C channel)
         {
-            Interlocked.Increment(ref s_closeCalls);
             // Getting a null would indicate an issue in harness
             if (channel == null)
             {
                 System.Diagnostics.Debugger.Break();
             }
```

Comment:
Nit: for the future -- consider a single helper that breaks to the debugger.  Then we can either #if DEBUG it or add logic to break only on certain conditions.  Not required for merge.